src/main/java/org/folio/circulation/resources/AbstractCirculationRulesEngineResource.java

```diff
@@ -29,14 +36,14 @@
 
 /**
  * The circulation rules engine calculates the loan policy based on
- * item type, loan type, patron type and shelving location.
+ * item type, loan type, patron type, location and institution.
```

Comment:
There is no need to mention the institution because only the location is passed in as a parameter and the location determines the institution.

Reply:
Yes, sure! Fixed.